src/core/toga/widgets/progressbar.py

```diff
@@ -19,26 +25,44 @@ def __init__(self, id=None, style=None, max=None, value=None, factory=None):
                 implementation of this class with the same name. (optional & normally not needed)
         """
         super().__init__(id=id, style=style, factory=factory)
+
+        self._running = False
         self._impl = self.factory.ProgressBar(interface=self)
 
         self.max = max
+        self.running = running
         self.value = value
-        self._running = False
 
     @property
-    def value(self):
-        """ The progress value
+    def running(self):
+        """
+        Returns:
+            True if the progress bar is running,
+            False if not
+        """
+        return self._running
+
+    @running.setter
+    def running(self, value):
+        self.enabled = bool(value or self.max)
+        self._running = value
+        self._impl.set_running(value)
```

Comment:
I can see you've replaced `start()`/`stop()` with the `running` property; I'm not sure if this is the right move. Using a property implies it's value-based, but in reality, it's a stateful operation. A call to start implies "Make this object do something", rather than assigning "give this object a state value of X".

Ultimately, I suppose the answer is "what is more Pythonic"... which then means we need to have a consistent interpretation of what *we* mean by "Pythonic".

Reply:
You're right; setting properties shouldn't have side effects. I think the argument for preferring `start()` / `stop()` is good. This does raise a couple of questions for me: 
-  Should `running` still exist as a getter only, or should it be removed? 
- Should the `running` be kept as a constructor parameter? 

In my opinion, it should be left as a getter *and* constructor so the state can be queried and the widget can be initialized in one line of code.

Reply:
I'd agree "running" should exist as a readonly - so you can interrogate current state. The next question is whether it should be `running` or `is_running`...

Reply:
You can't go wrong with `is_running`. On the other hand, which convention does Toga seem to prefer/use? There's `Widget.enabled`, `TextInput.readonly`, maybe others that use the former convention.

Reply:
Hey, I'm not saying we *are* consistent - I'm saying we *should be* consistent :-)

Effectively, we need a "style guide" for Toga API conventions. I've thrown out a quick Twitter poll to get a vague idea what people think.

Reply:
Ok - Broad feedback from twitter seems to be to a preference to keep things simple, and generally not have the `is_` prefix. There were a few counter arguments:

* Boolean values should have `is_` 
* Non-adjectives should have `is_`
* Irregular past participles should have `is_`
* Readonly boolean properties should have `is_`

From this, it's pretty clear we could very easily get into a minefield of *English* language lawyering.

The best idea I can think of at this point is use the `is_` prefix to signal something - and readonly boolean property status is the best thing to signal.  

So - `start()` is backed by a readonly `is_running`; but `enabled` and `readonly` are read/write properties.

Does that make sense as a general policy?

Reply:
`is_` = readonly boolean. Got it.